glide-core/src/client/mod.rs

```diff
@@ -474,6 +476,33 @@ impl Client {
         self.inflight_requests_allowed
             .fetch_add(1, Ordering::SeqCst)
     }
+
+    /// Replaces the current password with the provided ones.
+    /// If `re_auth` is true, the new password will be used to re-authenticate with the server.
```

Comment:
It isn't clear - is it going to re-auth all nodes (e.g primary and replicas/all cluster nodes)? please add better description